docs/install.md

```diff
@@ -1302,6 +1320,144 @@ deployment is in the `Failed` state, then one can look into the event(s) using
+`deployments.pmem-csi.intel.com` CRD is provided with the CA bundle(base64 encoded)
+used for signing the certificate.
+
+The provided [operator deployment yaml](/deploy/operator/pmem-csi-operator-webhook.yaml)
+uses `cert-manager` for providing the certificates and injecting the CA bundle into the CRD.
+
+Follow the below steps to install the operator with conversion webhook:
+
+- **Install the `cert-manger`**
+```console
+$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
+```
+
+- **Install the deployment CRD with webhook support**
+```console
+$ kubectl apply -f https://github.com/intel/pmem-csi/devel/deploy/crd/pmem-csi.intel.com_deployments_webhook.yaml
+```
+
+- **Install the PMEM-CSI operator**
+```console
+$ kubectl apply -f https://github.com/intel/pmem-csi/raw/devel/deploy/operator/pmem-csi-operator-webhook.yaml
+```
+
+With the above setup, the deprecated deployment API version will automatically
+gets converted to an appropriate new version. The below example shows that the
+`nodeResources` and `controllerResources` of `v1alpha1` API are converted to
+equivalent `v1beta1` API fields, i.e.,`nodeDriverResources` and
+`controllerDriverResources` respectively.
+
+```console
+$  kubectl get crd deployments.pmem-csi.intel.com -o jsonpath='{.spec.conversion}{"\n"}'
+{"strategy":"Webhook","webhook":{"clientConfig":{"caBundle":"LS0tLS1CRUdJTiBDRV...","service":{"name":"pmem-csi-operator-service","namespace":"default","path":"/convert","port":9443}},"conversionReviewVersions":["v1beta1","v1alpha1"]}}
+
+kubectl create -f - << EOF
+apiVersion: pmem-csi.intel.com/v1alpha1
+kind: Deployment
+metadata:
+  name: pmem-csi.intel.com
+spec:
+  deviceMode: lvm
+  nodeResources:
+    limits:
+      cpu: "500m"
+      memory: "250Mi"
+  controllerResources:
+    limits:
+      cpu: "400m"
+      memory: "150Mi"
+EOF
+deployment.pmem-csi.intel.com/pmem-csi.intel.com created
+
+$ kubectl get deployments.pmem-csi.intel.com pmem-csi.intel.com -o jsonpath='{.apiVersion}{"\n"}{.spec}{"\n"}'
+pmem-csi.intel.com/v1beta1
+{"controllerDriverResources":{"limits":{"cpu":"400m","memory":"150Mi"}},"deviceMode":"lvm","nodeDriverResources":{"limits":{"cpu":"500m","memory":"250Mi"}}}
+
+$ kubectl get deployments.v1alpha1.pmem-csi.intel.com pmem-csi.intel.com -o jsonpath='{.apiVersion}{"\n"}{.spec}{"\n"}'
+pmem-csi.intel.com/v1alpha1
+{"controllerResources":{"limits":{"cpu":"400m","memory":"150Mi"}},"deviceMode":"lvm","nodeResources":{"limits":{"cpu":"500m","memory":"250Mi"}}}
+
+```
+
+**NOTE**: Conversion webhook is not yet supported by the operator installed using the
+[Operator Lifecycle Manager](https://operatorhub.io/operator/pmem-csi-operator).
+If the operator is deployed with no conversion webhook then it will not
+support the conversion of deprecated API fields as described above.
+In that case, the user has to manually migrate their existing driver deployments
+to use the latest operator API.
```

Comment:
Huh? I thought the apiserver will convert internally when no webhook is configured. So why does a user "have to manually migrate their existing driver deployments"?

Reply:
Hmm, "in that case" is not clear. It doesn't refer to "deployed with no conversion webhook" (my initial understanding) but rather "conversion of deprecated API fields". Will fix...